docs/source/tutorials/how_to_test.ipynb

```diff
@@ -488,7 +526,7 @@
vj4AwMZf3ImNv7gz9nZ+yxcfVWf/Bfx68/349eb7E+2bKI6N11zZ7UNInb/NBpobTaITBvr68KdtT+GRp3a5j8Pbq+jtC/jNfQ/hN/dF5nKqnNa3OnS5pgn6CgZ+dvdm/CyiLSWi7qrXZqfRkoVFuKpmJZH1ANYDwNIVK/HWC14BADjmqAUdPTia8aKTnwdHKeiahjlDg9h36HDoegN9fVi77JjQ515wwrG4/MJXwrQsLFk4v6mM0pzhQbznktfi4JHxRMcPAKcdf2zkc3938aux47k9ifdNRDOi2uzB/n6sWbq4m4eGy85/OV7wxNOV41l1THPHs/4N5+PJ3WMx1xacsW5t5LPvu/QvsHvv/qZen4iyQfzj4nbkBUTOAnCNUurP3cdXAYBS6p+ithkdHVWbN/NbNxHl0qy79cU2m4hyLLLNTqN3xX8AOEFEjhWRPgBvBvDTFF6XiIiIiCiRjpdbKKUsEXkPgFsB6ACuU0o92OnXJSIiIiJKKpXeFUqpnwP4eRqvRURERETUqo7XJCchIkcAPAvgUJt2uQjA3jbtax7ad1zt2hfPL752HVM7f09ZO78snlue3pcDSqlT2rD/3HDb7EfA905cWfvMt3tfWTy/LL43AbbZae8r7Pyi22ylVOb+AdgMYEM799fGfbXzuNqyL55fV46pZ9+fWTy3PL0v2/n3zMs/75z53kn33Hh+Xfs9Zer8snhuWXxfRp1fvXPO8lR2N3X7ACK087iyeI48v3T3027tOK4snluvvy97RRZ/t73+3uH5pbufdmObne6+mpLZIFkplcU/eluPK4vnyPNLdz/t1o7jyuK59fr7sldk8Xfb6+8dnl+6+2k3ttnp7qtZWQ2SN2R8f1nD88u3Xj6/2XhuvXzOUTpxzr38e+zlcwN4fnnWy+cGhJ9f5DlnsuMeEREREVE3ZTWTTERERETUNT0VJIvIG0VEicjzu30snSIi4w2ev1NERtM6nnYRkRUi8hMReUxEtonI59wZGqPWf7+IDKV5jK1q9LfLq9nwuQN697PXTbPhvdOr7xu22fnGz168z15PBckALgPwW5Snvo5NRPTOHA7FISIC4EcAfqyUOgHA8wCMAPh4nc3eDyBXDW4P4+eOkuJ7J4fYZvcEfvZi6JkgWURGAJwN4B1w/+gico6I3CUiN4rIQyJyrYho7nPjIvIxEdkE4KzuHXnz3PO62ff4CyLy9i4eUqvOA1BUSv0bACilbABXAvhbERkWkc+IyJ9E5H4Rea+IXAFgGYA7ROSOLh5300RkRERuF5F73HN6vbt8jYg8LCJfFZEHReQ2ERns9vE2Mps+d0BPfva6Zja9d3rwfcM2O6dtNsDPXjOfvZ4JkgG8AcAtSqlHAewXkTPc5WcC+ACAUwEcB+Aid/kwgAeUUi9SSv027YOlKicD2OJfoJQ6DOBpAO8EcCyA05VSpwG4Xin1eQDPADhXKXVu2gfboiKANyqlzgBwLoDPulkZADgBwBeVUicDOAjg4u4cYlPeAH7uKJk3gO+dvGKbXZbHNhvgZy+2XgqSLwPwXffn77qPAeAPSqnt7jfd7wB4qbvcBvDDdA+RIgiAsGFWBMDLAVyrlLIAQCm1P80D6wAB8AkRuR/ArwAsB7DEfe4JpdR97s9bAKxJ/eiax88dJcX3Tn6xzS7LY5sN8LMXm9HtA2gHETkK5ds/p4iIAqCj/AH+OWo/yN7jovtGyCML1V9wBrp1IG3yIALfwEVkLoCVALYjvDHOq78CcDSAP1NKmSLyJGb+ftO+9WwAmb51Nws/d0Dvffa6Yha+d3rtfcM2uyxXbTbAzx6a/Oz1Sib5TQC+qZRarZRao5RaCeAJlL8FnSkix7q1NZeiXKied08BOElE+kVkHoBXdvuAWnQ7gCER+Rug0jHgswC+DuA2AO8WEcN9bqG7zREAc9I/1JbNAzDmNrbnAljd7QNqwWz73AG999nrltn23um19w3b7PziZ68JvRIkXwbgxsCyHwJ4C4DfAfgkgAdQfiME18sNt9GZVkrtAHADgPsBXA/g3q4eWItUeUabNwK4REQeA/AoynVgVwP4Gsp1bveLyB9R/psC5RlyfpGXTiDe3w7lv9eoiGxGOUOxtasH1ppZ8bkDevez10Wz4r3Tq+8bttm5xs9eM/vp5Rn3ROQcAP9VKXVhlw+lLUTkBQC+qpQ6s9vHQs2ZTX+7XvvcAbPr79dNvfbe4fsmv2bb346fvXC9kknueSLybpQL6T/S7WOh5vBvl2/8+1ESfN/kF/92+dbOv19PZ5KJiIiIiJJgJpmIiIiIKIBBMlGbichKEbnDnY3pQRF5n7t8oYj8UkQec/9f4C5/lYhscWdz2iIi5/n29XER2SEN5qAnIqJk2GZTFJZbELWZiCwFsFQpdY+IzEF5kPk3AHg7gP1KqU+KyIcBLFBK/YOInA7gOaXUMyJyCoBblVLL3X29GOUhbB5TSo1043yIiHoZ22yKwiCZqMNE5CcAvuD+O0cptdttlO9USq0LrCsA9gJYppSa9i0fZ4NLRNR5bLPJw3ILog4SkTUATgewCcASpdRuAHD/XxyyycUA7vU3tkRElA622eTXE9NSE2WRiIygPEj7+5VSh8sJh7rrnwzgUwDOT+HwiIjIh202BTGTTNQBIlJAubG9Xin1I3fxc+4tO68Gbsy3/gqUZzf6G6XUtrSPl4hoNmObTWEYJBO1mVuj9q8AHlZK/bPvqZ8CeJv789sA/MRdfz6AnwG4Sil1d4qHSkQ067HNpijsuEfUZiLyUgC/AfAnAI67+GqUa9xuALAKwNMALlFK7ReRjwC4CsBjvt2cr5QaE5H/DeAtAJYBeAbA15RS16RyIkREswDbbIrCIJmIiIiIKIDlFkREREREAQySiYiIiIgCGCQTEREREQUwSCYiIiIiCmCQTEREREQUwCCZiIiIiCiAQTIRERERUQCDZCIiIiKigP8PPndLwjXwTO4AAAAASUVORK5CYII=\n",
```

Comment:
It is because ``share_known_cases`` does not play well with the testing models which will be resolved in #76. As you can see [there](https://sid-dev--76.org.readthedocs.build/en/76/tutorials/how_to_test.html), the plots will be fine.

It is because ``share_known_cases`` is a series full of 0 by default which will overwrite any tests assigned by the testing models.